tests/client/disco/containers/TestInstallButton.js

```diff
@@ -22,13 +13,34 @@ import {
   UNKNOWN,
 } from 'disco/constants';
 
+const AddonManager = addonManager.AddonManager;
+
 
 describe('<InstallButton />', () => {
-  function renderButton(props) {
-    return renderIntoDocument(<InstallButton { ...props } />);
+  let sandbox;
+
+  function stubAddonManager({ getAddon = Promise.resolve() } = {}) {
+    const instance = sinon.createStubInstance(AddonManager);
```

Comment:
The instance is created when the component mounts which is done by react. I think we'd need a proper do DI setup to do that.